types/magicmirror-module/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for non-npm package magicmirror-module 2.16
+// Project: https://magicmirror.builders/
+// Definitions by: Jalibu <https://github.com/jalibu>
+//                 MichMich <https://github.com/MichMich>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.7
+
+declare namespace Module {
+    /* tslint:disable:no-unnecessary-generics */
+    function register<T>(moduleName: string, moduleProperties: ModuleProperties<T>): void;
```

Comment:
```suggestion
    function register<T>(moduleName: string, moduleProperties?: ModuleProperties<T>): void;
```
docs explicitely shows example without options:
https://docs.magicmirror.builders/development/core-module-file.html#the-core-module-file 
which is correct?